hw1/task2/bash.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+
+def wc(first: str, *args: str):
+    def wc_for_one(filepath: str):
+        size = os.path.getsize(filepath)
+        f = open(filepath)
+        words = 0
+        lines = 0
+        for line in f:
+            lines += 1
+            words += len(line.split())
+        f.close()
+        return {"newlines": lines, "words": words, "bytes": size}
+
+    acc = {"newlines": 0, "words": 0, "bytes": 0}
```

Comment:
We should try to keep the identifiers more understandable. I guess `acc` stands for `accumulated`, but its hard to understand initially. I think a more obvious name like `total` is easier to understand.